<ui version="4.0" >
 <class>MainForm</class>
 <widget class="Q3MainWindow" name="MainForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>657</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>56</height>
   </size>
  </property>
  <property name="font" >
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="mouseTracking" >
   <bool>true</bool>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string comment="Prototype Mesomodel" >Mezzo version 0.51</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="canvas_qt4.qrc" >:/images/mime.png</iconset>
  </property>
  <property name="windowIconText" >
   <string>Mezzo</string>
  </property>
  <property name="toolTip" >
   <string/>
  </property>
  <property name="rightJustification" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>54</y>
     <width>799</width>
     <height>603</height>
    </rect>
   </property>
   <widget class="QWidget" native="1" name="widget_2" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>610</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="status_label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>578</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>uninitialized</string>
    </property>
   </widget>
   <widget class="QWidget" native="1" name="simprogress_widget" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>578</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QLCDNumber" name="LCDNumber" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>Simulated time elapsed</string>
     </property>
     <property name="numDigits" >
      <number>8</number>
     </property>
     <property name="segmentStyle" >
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0" >
      <double>0.000000000000000</double>
     </property>
     <property name="intValue" stdset="0" >
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="TextLabel1" >
     <property name="geometry" >
      <rect>
       <x>190</x>
       <y>0</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Times New Roman</family>
      </font>
     </property>
     <property name="text" >
      <string>Progress:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
    <widget class="Q3ProgressBar" name="progressbar" >
     <property name="geometry" >
      <rect>
       <x>260</x>
       <y>5</y>
       <width>120</width>
       <height>12</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="TextLabel12" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>0</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <family>Times New Roman</family>
      </font>
     </property>
     <property name="text" >
      <string>Time:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="mouse_label" >
    <property name="geometry" >
     <rect>
      <x>600</x>
      <y>578</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text" >
     <string>No mouse event</string>
    </property>
   </widget>
   <widget class="QLabel" name="Canvas" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>570</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy>
      <hsizetype>7</hsizetype>
      <vsizetype>7</vsizetype>
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="palette" >
     <palette>
      <active>
       <colorrole role="WindowText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>170</red>
          <green>170</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>170</red>
          <green>170</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>170</red>
          <green>170</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase" >
        <brush brushstyle="SolidPattern" >
         <color alpha="255" >
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="mouseTracking" >
     <bool>true</bool>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground" >
     <bool>false</bool>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth" >
     <number>0</number>
    </property>
    <property name="midLineWidth" >
     <number>0</number>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="scaledContents" >
     <bool>false</bool>
    </property>
    <property name="alignment" >
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
    <property name="margin" >
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="Q3ToolBar" name="toolBar1" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>338</width>
     <height>32</height>
    </rect>
   </property>
   <property name="horizontallyStretchable" >
    <bool>true</bool>
   </property>
   <property name="verticallyStretchable" >
    <bool>true</bool>
   </property>
   <property name="label" >
    <string>Tools</string>
   </property>
   <addaction name="openmasterfile" />
   <addaction name="savescreenshot" />
   <addaction name="saveresults" />
   <addaction name="separator" />
   <addaction name="run" />
   <addaction name="breakoff" />
   <addaction name="quit" />
   <addaction name="zoomin" />
   <addaction name="zoomout" />
   <addaction name="zoombywin" />
   <addaction name="linkhandlemark" />
   <addaction name="separator" />
  </widget>
  <widget class="Q3ToolBar" name="Toolbar2" >
   <property name="geometry" >
    <rect>
     <x>338</x>
     <y>0</y>
     <width>79</width>
     <height>32</height>
    </rect>
   </property>
   <property name="label" >
    <string>Toolbar</string>
   </property>
   <addaction name="parametersdialog" />
   <addaction name="inspectdialog" />
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="EditMenu" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="parametersdialog" />
    <addaction name="loadbackground" />
    <addaction name="inspectdialog" />
   </widget>
   <widget class="QMenu" name="fileMenu" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="openmasterfile" />
    <addaction name="savescreenshot" />
    <addaction name="separator" />
    <addaction name="saveresults" />
    <addaction name="quit" />
   </widget>
   <addaction name="fileMenu" />
   <addaction name="EditMenu" />
  </widget>
  <action name="openmasterfile" >
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/fileopen2.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open Master file...</string>
   </property>
   <property name="iconText" >
    <string>&amp;Open Master file...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="savescreenshot" >
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/background.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save Snapshot</string>
   </property>
   <property name="iconText" >
    <string>Save Snapshot</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="quit" >
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/bt_stop.png</iconset>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
   <property name="iconText" >
    <string>Quit</string>
   </property>
   <property name="toolTip" >
    <string>Quit</string>
   </property>
   <property name="statusTip" >
    <string>Quit</string>
   </property>
  </action>
  <action name="run" >
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/bt_play.png</iconset>
   </property>
   <property name="iconText" >
    <string>Run</string>
   </property>
   <property name="whatsThis" >
    <string>Run the model</string>
   </property>
  </action>
  <action name="breakoff" >
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/bt_pause.png</iconset>
   </property>
   <property name="iconText" >
    <string>Pause</string>
   </property>
   <property name="statusTip" >
    <string>Pause simulation</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="zoomin" >
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/zoom_in.png</iconset>
   </property>
   <property name="iconText" >
    <string>Zoom in</string>
   </property>
  </action>
  <action name="zoomout" >
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/zoom_out.png</iconset>
   </property>
   <property name="iconText" >
    <string>Zoom out</string>
   </property>
  </action>
  <action name="viewSet_ParametersAction" >
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/icon-tool.png</iconset>
   </property>
   <property name="iconText" >
    <string>&amp;Set Parameters</string>
   </property>
  </action>
  <action name="parametersdialog" >
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/icon-tool.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Parameters</string>
   </property>
   <property name="iconText" >
    <string>&amp;Parameters</string>
   </property>
  </action>
  <action name="loadbackground" >
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/dvi.png</iconset>
   </property>
   <property name="text" >
    <string>Load &amp;Background</string>
   </property>
   <property name="iconText" >
    <string>Load &amp;Background</string>
   </property>
   <property name="toolTip" >
    <string>Load Background&amp;</string>
   </property>
  </action>
  <action name="saveresults" >
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>Save Results</string>
   </property>
  </action>
  <action name="inspectdialog" >
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/inspect.png</iconset>
   </property>
   <property name="text" >
    <string>Inspect</string>
   </property>
  </action>
  <action name="zoombywin" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/file_explore.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom window</string>
   </property>
  </action>
  <action name="linkhandlemark" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="canvas_qt4.qrc" >:/images/penmark.png</iconset>
   </property>
   <property name="text" >
    <string>linkhandlemark</string>
   </property>
   <property name="toolTip" >
    <string>Show link handlers</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Q3ProgressBar</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3ProgressBar</header>
  </customwidget>
  <customwidget>
   <class>Q3ToolBar</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
  <customwidget>
   <class>Q3MainWindow</class>
   <extends>QWidget</extends>
   <header>q3mainwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global" >qlcdnumber.h</include>
  <include location="global" >qbitmap.h</include>
  <include location="global" >qpainter.h</include>
  <include location="global" >qpixmap.h</include>
  <include location="global" >qtimer.h</include>
  <include location="global" >qpen.h</include>
  <include location="global" >string</include>
  <include location="global" >qstatusbar.h</include>
 </includes>
 <resources>
  <include location="canvas_qt4.qrc" />
 </resources>
 <connections/>
</ui>
